### PR TITLE
made some classes

### PR DESCRIPTION
**Pull request by do-ruth**
@quietHamster

I made a card class and a deck class so far!
I tested both to see if I could make a card and show a deck and shuffle the deck and it seems to work for me.
Physically going through this myself helped me understand a lot more about classes and OOP.

Next steps are to make the hands for the players and start on the main method for the game itself!

### CODE REVIEW

File: src/Card.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+    }
+
+    /**
+     * This returns the face and suit of the Card object as a string
+     * @return
+     */
+    public String toString() {
+
+        return faceName + " of " + suit;
```
Comment (quietHamster) [inline src/Card.java:26]: I recommended Not to use "+" with String. It is best practice to use String.concat() method.
I would write:
`return faceName.concat(" + ").concat(suit);`

File: src/Deck.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        }
+    }
+
+    /**
+     * Printing the deck
+     */
+    public void displayDeck() {
+
+        for (Card card : deck)
+            System.out.println(card);
```
Comment (quietHamster) [inline src/Deck.java:41]: what is the console output for this ?
should it be  `System.out.println(card.toString()); ` ?

File: src/Deck.java [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+    }
+
+    public void shuffle() {
+
+        Random mrRandom = new Random();
+        int currentCard = 0;
+        int first = 0;
+        while (currentCard < deck.length) {
+            int second = mrRandom.nextInt(52);
```
Comment (quietHamster) [inline src/Deck.java:51]: what can ensure that no duplicates here? there is chance that `mrRandom.nextInt(52)` would return duplicates.

File: src/Deck.java [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+            System.out.println(card);
+
+    }
+
+    public void shuffle() {
+
+        Random mrRandom = new Random();
+        int currentCard = 0;
+        int first = 0;
+        while (currentCard < deck.length) {
```
Comment (quietHamster) [inline src/Deck.java:50]: can you explain this shuffle algorithm in plain English? what happen inside this `while loop`?

**Review by quietHamster — COMMENTED** [review]
Good approach!

File: src/Deck.java (hunk 2 above)
Comment (do-ruth) [reply src/Deck.java:41]: everything worked fine for me, but it might be good practice for me to use the ".toString" 

my output was 52 different cards 
"Jack of Hearts
8 of Spades
...
etc."

File: src/Deck.java (hunk 3 above)
Comment (do-ruth) [reply src/Deck.java:51]: yes, there might be duplicates but in the way that I intended to use this, I just wanted to switch the cards around with each other. So even if the random returned a duplicate, the cards would just switch around. For example, I'm switching the card in index 0 with index 32. Then the next index 1 could switch with index 32 again but the cards are not repeating and will still end up shuffled.

File: src/Deck.java (hunk 4 above)
Comment (do-ruth) [reply src/Deck.java:50]: Create random
Current card is used to track how many cards we've been through. Just going to the end of the deck.
"first" is the first card and "second" is the second card. these two will be swapping
while the current card is less than the deck length, then the "first" card is the first of the pair I will use to swap. So I set a temp card equal to the "first" card. Then I used the random to call a "second" card at that random number index. I set the "first" card to equal the "second" and then the "second" equal to the temp (which was the first card).

I'm just swapping cards two at a time with each other. The end result is still a shuffled deck.

File: src/Deck.java (hunk 3 above)
Comment (quietHamster) [reply src/Deck.java:51]: good point

File: src/Deck.java (hunk 4 above)
Comment (quietHamster) [reply src/Deck.java:50]: nice approach,
